docs/source/breakpoints.rst

```diff
@@ -28,12 +28,31 @@ In the provided example, the debugger will set a breakpoint at the address `0x10
 
 When the program reaches the breakpoint, it will pause. You can enable and disable breakpoint `bp` with the `bp.enable()` and `bp.disable()` functions, respectively.
 
+Relative to What?
+-----------------
+
+By default, the memory access in libdebug is done using an hybrid addressing mode. This means that libdebug will try to resolve the address as an absolute address first. If the address is not found, libdebug will try to resolve the address as a relative address, using as base the one of the binary. In this case, a warning will be printed. You can force the addressing mode by using the following syntax:
+
+.. code-block:: python
+
+    from libdebug import debugger
+
+    d = debugger("./test_program")
+
+    d.run()
+
+    bp = d.breakpoint(0x10ab, file="libc")
+
+    d.cont()
+
+In this case, the breakpoint will be set at the address ``0x10ab`` relative to the ``libc`` memory page. Other than existing pages, you can specify ``absolute`` as the file to force the absolute addressing mode.
+
```

Comment:
you can also specify "binary" or the name of the binary file to tell libdebug to resolve the symbol w.r.t. the binary's mem maps

Reply:
Resolved in subsequent commit.